test/e2e/components/item.js

```diff
@@ -0,0 +1,13 @@
+const { By } = require('selenium-webdriver');
+
+class Item {
+    constructor(value, text, isSelected, webElement) {
+        this.value = value;
+        this.text = text;
+        this.isSelected = isSelected;
+        this.webElement = webElement;
```

Comment:
webelement nodig?

Reply:
Is inderdaad overbodig geworden. Ik haal het weg.

Reply:
Blijkt toch nodig te zijn omdat het er nog staat?

Reply:
Changes zijn nog niet gepushed.